src/main/resources/site/filters/security-headers.js

```diff
@@ -88,13 +89,17 @@ exports.responseFilter = function (req, res) {
             headers["X-XSS-Protection"] = h;
         }
 
-        if (siteConfig.xContentTypeOptions) {
+        if (siteConfig.xContentTypeOptions.value) {
```

Comment:
Both siteConfig.xContentTypeOptions and siteConfig.permissionsPolicy are item-sets so it could be possible for these to be undefined if the editor deletes the items. Then you'll likely get an error trying to check the value or policy properties.

Might be safer to use optional chaining here and in the cases below as well. :)